eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/configuration/SwaggerConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ebi.eva.accession.ws.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfiguration {
+
+    @Bean
+    public Docket productApi() {
+        return new Docket(DocumentationType.SWAGGER_2)
+                .select()
+                .apis(RequestHandlerSelectors.basePackage("uk.ac.ebi.eva.accession.ws.rest"))
+                .paths(PathSelectors.any())
+                .build();
+    }
```

Comment:
More details need to be added to this configuration:

```java
     private ApiInfo getApiInfo() {
         return new ApiInfoBuilder()
                 .contact(new Contact("Europe Variation Archive - EMBL-EBI", "https://www.ebi.ac.uk/eva", null))
                 .license("Apache 2.0")
                 .licenseUrl("http://www.apache.org/licenses/LICENSE-2.0")
                 .termsOfServiceUrl(null)
                 .title("EVA variant identifiers API")
                 .description("API to retrieve information about SubSNP (ss) and RefSNP (rs) variant identifiers")
                 .version("0.1")
                 .build();
     }
```

Which can then be added with

```java
new Docket(...).apiInfo(getApiInfo())
```